src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -65,8 +65,28 @@ export function createOVOptions(options = []) {
   }
 }
 
+/**
+ * Example of the option like
+ * @param {AuthenticatorOption[]} options
+ * @param {( AuthenticatorEnrollment[] || Authenticator[] )} authenticators
+ */
+function updateCustomAppOptions(options = []) {
+  // update custom app options such that methodType value is sent to backend.
+  const customAppItems = options.filter((option) => option.relatesTo.key === AUTHENTICATOR_KEY.CUSTOM_APP);
+  customAppItems.forEach((customAppItem) => {
+    const methodTypeObj = customAppItem?.value?.form?.value?.find(v => v.name === 'methodType');
+    if (methodTypeObj) {
+      // set the methodType field to be required and immutable in our UI,
+      // so it is always sent to the backend.
+      methodTypeObj.required = true;
```

Comment:
As this [validation](https://github.com/okta/okta-core/blob/master/components/platform/idx/impl/src/main/java/com/okta/platform/services/idx/pipeline/services/IdxStateTokenTransactionManagerImpl.java#L1598) is failing in backend and not checking for autoChallenge we need to pass methodType to backend to match with requiredFields.